### ISSUE TITLE
Fix GKE Install docs

### ISSUE THREAD

**Issue by geeknoid** [istio/istio.github.io#1786]
From an email:

I am running the steps on: https://preliminary.istio.io/docs/setup/kubernetes/quick-start-gke-dm/
However, the link to get the Istio GKE Deployment Manager is broken.
That is a stopper for the doc, meaning me, and users, cannot continue.

@salrashid123

**Comment by geeknoid** [comment]
@rcaballeromx

**Comment by andraxylia** [comment]
Duplicate of istio/istio#7049

We need to wait for the release to be created before updating the .jinja file.

**Comment by andraxylia** [comment]
why did you re-open?

**Comment by geeknoid** [comment]
I think we need a doc review as well.

**Comment by salrashid123** [comment]
yeah, the link in the docs points back to a DM template that is verison specific

for example, the current link on istio.io and on ```preliminary.istio.io```  both point to a DM template for ```release-0.8```

eg, the docs have a link like this in it to launch:
```
"https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/release-0.8/install/gcp/deployment_manager/istio-cluster.jinja"
```

its going to be slightly werid update on this and parts will span this pr and the one i'm working on for the template...what i can do is test the changes first and then commit  the PR for the ```release-1.0```  'just using' ```release-1.0``` in the URL.

**Comment by geeknoid** [comment]
Oh, in the .md file, the
https://raw.githubcontent.com/istio/istio/release-0.8 string should be
replaced with {{<github_file >}}. This will create a link that always
points to the right branch for the repo.

**Comment by rcaballeromx** [comment]
I am already editing this doc and can make the changes. Just let me know the correct links and I'll replace them.

**Comment by rcaballeromx** [comment]
@geeknoid Can you give me the complete {<github_file >}} link I should use?

**Comment by andraxylia** [comment]
The release 1.0 is not yet published.
